omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -193,156 +197,115 @@
         </p>
     </div>
 
-    <section>
-        <div class="small-text section-container">
+    <section id="vitals" sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.vitals.label">
+        <div class="section-container">
             <table>
                 <thead>
                     <tr>
                         <td style="font-weight:bold">
-                            <uimessage code="pihcore.mostRecentVitals.label"/>
+                            <uimessage code="pihcore.vitals.latest.measurement.label"/>
                         </td>
                         <td style="font-weight:bold">
                             <uimessage code="zl.date"/>
                         </td>
                         <td style="font-weight:bold">
-                            <uimessage code="mirebalais.vital.results"/>
+                            <uimessage code="pihcore.vitals.latest.results.label"/>
                         </td>
                     </tr>
                 </thead>
                 <tbody>
                     <tr>
                         <td>
-                            <uimessage code="pihcore.vitals.systolic.bp.title"/>
+                            <uimessage code="mirebalais.vitals.bloodPressure.title" />
                         </td>
                         <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:SYSTOLIC BLOOD PRESSURE').obsDatetime"/>
-                            </p>
+                            <lookup expression="fn.latestObs('PIH:SYSTOLIC BLOOD PRESSURE').obsDatetime"/>
```

Comment:
Assume that diastolic and systolic will always be recorded together.

Reply:
@brandones , I had done this approach before but then I realized the info might be misleading in cases where they are not done together. maybe there could be another approach where it evaluates and ensures that the obs_datetime is the same?

Reply:
Yeah, while that would be great, it would be an extreme corner case -- it would mean that someone screwed up data entry pretty badly. But you're right, it is misleading in the case of that error. Given the constraints of our tools, I think we have to be okay with that.